pkg/trace/sampler/signature_test.go

```diff
@@ -6,6 +6,8 @@
 package sampler
 
 import (
+	"hash/fnv"
+	"k8s.io/apimachinery/pkg/util/rand"
```

Comment:
Please pay attention to the import grouping. We use 3 import groups, each separated by an empty line:
1. Standard library
2. Datadog Agent
3. External